src/test/ui/internal/internal-unstable-const.stderr

```diff
@@ -1,13 +1,17 @@
-error[E0658]: `transmute` is not allowed in constant functions
-  --> $DIR/internal-unstable-const.rs:11:14
+error: const-stable function cannot use `#[feature(const_fn_floating_point_arithmetic)]`
+  --> $DIR/internal-unstable-const.rs:10:5
    |
-LL |     unsafe { std::mem::transmute(4u32) }
-   |              ^^^^^^^^^^^^^^^^^^^^^^^^^
+LL |     1.0 + 1.0
+   |     ^^^^^^^^^
+   |
+help: if it is not part of the public API, make this function unstably const
+   |
+LL | #[rustc_const_unstable(feature = "...", issue = "...")]
+   |
+help: otherwise `#[rustc_allow_const_fn_unstable]` can be used to bypass stability checks
+   |
+LL | #[rustc_allow_const_fn_unstable(const_fn_floating_point_arithmetic)]
```

Comment:
Oh fancy, it's a completely different error that previously... @oli-obk is that expected? Seems like a bug that the fancy error was not already used before.

Anyway, the new test looks good!

Reply:
yea, the way I unstabilized transmute was not a point of pride